pkg/build/api/validation/validation.go

```diff
@@ -64,14 +64,15 @@ func ValidateBuildConfig(config *buildapi.BuildConfig) field.ErrorList {
 	fromRefs := map[string]struct{}{}
 	specPath := field.NewPath("spec")
 	triggersPath := specPath.Child("triggers")
+	buildFrom := buildutil.GetImageStreamForStrategy(config.Spec.Strategy)
```

Comment:
yeah I guess we do when we update the imagechangetrigger status w/ the latest imageid.

what if we add a conversion that removes default imagechangetriggers that would cause a validation failure?  they are serving no purpose anyway.